src/media/shortcodes/class-wpml-page-builders-media-shortcodes-update.php

```diff
@@ -41,7 +45,13 @@ public function translate( $post ) {
 
 		$this->media_usage->update( $element->get_source_element()->get_id() );
 
-		wp_update_post( $post );
+		$tag_ids = wp_get_post_tags( $post->ID, array( 'fields' => 'ids' ) );
```

Comment:
Please add a comment to explain the reason for this code otherwise it does not make sense (include a link to the related WP Trac ticket and YT ticket).

Reply:
Done.